Docs payments (#6677)
vuestorefront/vue-storefront: packages/core/docs/integrate/payment.md

:::

### Saving cards for recurring payments

Payment service providers like Adyen allow storing user credit cards for future payments. They store them inside their database, but you have to create a mechanism that generates unique User IDs that others can't see or manipulate.

For example, you shouldn't create an endpoint that returns saved cards based on data provided by the user because, with enough (bad) luck, someone can guess this ID and download the card information of another user.

Instead, you should use data that identifies if the user is who they say they are (such as their access token) to download saved cards.

### Strong Customer Authentication

In [European Economic Area](https://en.wikipedia.org/wiki/European_Economic_Area), each bank has to perform [Strong Customer Authentication](https://en.wikipedia.org/wiki/Strong_customer_authentication) such as [3DS1 or 3DS2](https://www.tokenex.com/blog/what-is-3-d-secure-authentication-and-why-do-i-need-it). This requirement means that users have to verify the payment by entering an SMS Code from the bank, in the bank's application, or a similar authorization mechanism.

> This verification is sometimes skipped in PSPs sandbox mode to make development faster. Before going into production, make sure to test both 3DS1 and 3DS2.

Reviewer: ```suggestion
Before using your integration in production, make sure to test both 3DS1 and 3DS2.
```
